### PR TITLE
Launcher in French / Launcher en Français

### PR DESCRIPTION
**Pull request by GaelHF**
I added and translated the en_US.toml file in French to make fr_FR.toml.

### CODE REVIEW

File: app/assets/lang/fr_FR.toml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+newReleaseTitle = "Mise à jour disponible !"
+newPreReleaseTitle = "Pré mise à jour disponible !"
+downloadingButton = "Téléchargement...."
+downloadButton = 'Download from GitHub<span style="font-size: 10px;color: gray;text-shadow: none !important;">Close the launcher and run the dmg to update.</span>'
+latestVersionTitle = "Vous avez la dernière version."
+checkForUpdatesButton = "Rechercher des mises à jours"
+checkingForUpdatesButton = "Recherche de mises à jours..."
+
+[js.uibinder.startup]
+fatalErrorTitle = "Fatal Error: Unable to Load Distribution Index"
```
Comment (dscalzi) [inline app/assets/lang/fr_FR.toml:267]: still english

**Comment by dscalzi** [comment]
@GeekCornerGH what do you think? I'm wondering if we should add french as well since most forks are french users.

**Comment by Vaxs02** [comment]
@GeekCornerGH stp dit oui

**Comment by GeekCornerGH** [comment]
> @GeekCornerGH what do you think? I'm wondering if we should add french as well since most forks are french users.

That's a good question. I don't think adding i18n should be in the scope of the project, but it's your project, so do whatever you want

**Review by GeekCornerGH — CHANGES_REQUESTED** [review]
Review (biggest one I've ever done yet) is finally done, merry Christmas🎄🎉 
To be honest, there were quite some grammar mistakes, while some sentences weren't making sense at all. And I probably missed some.
Have you done the translation through Google Translate by any luck?

**Review by sheazywi — APPROVED** [review]

File: app/assets/lang/fr_FR.toml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+noSelection = "Aucune serveur sélectioner"
+loading = "Chargement.."
+
+[js.landing.serverStatus]
+server = "SERVEUR"
+offline = "HORS LIGNE"
+players = "JOUEURS"
+
+[js.landing.systemScan]
+checking = "Anylse les informations du système.."
```
[the remark this answers is not in the document]
Comment (Aeris1One) [reply app/assets/lang/fr_FR.toml:166]: Il y a une faute de frappe sur `Analyse`

[the remark this answers is not in the document]
Comment (GeekCornerGH) [reply app/assets/lang/fr_FR.toml:166]: Thanks for the catch, fixed

**Pull request closed without merging**